src/contract/utils.ts

```diff
@@ -107,3 +110,12 @@ export function processSpecEntryStream(buffer: Buffer) {
   }
   return res;
 }
+
+export async function getAccount<T>(
```

Comment:
do we need to export globally or can we keep it privately w/in `contract`?

Reply:
it is private! `utils` is not exported in `index.ts`